app/views/users/show.html.erb

```diff
@@ -1,9 +1,71 @@
 <section class="user-section">
-  <h2><%= "Name: #{@user.name}" %></h2>
+  <h2>
+    <%= @user.name %> 
+    <% unless @user == current_user %>
```

Comment:
**[REQUIRED]**
* Let's remove all the logic from the views. Views related logic should be place in the helpers. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/section/no_logic_in_helpers_requirement.png) from Pathwright)
Resources:
[Where Do I Put My Code?](http://codefol.io/posts/Where-Do-I-Put-My-Code)
[The Beginner's Guide to Rails Helpers](https://mixandgo.com/learn/the-beginners-guide-to-rails-helpers)

Reply:
The only 'if' statements are to decide the appearance of elements
There is no logic in the views
You did not understand our code probably